packages/contracts/contracts/L1/ValidatorPool.sol

```diff
@@ -43,9 +43,9 @@ contract ValidatorPool is ReentrancyGuardUpgradeable, Semver {
     address public immutable TRUSTED_VALIDATOR;
 
     /**
-     * @notice The minimum bond amount. Can be updated via upgrade.
+     * @notice The required bond amount. Can be updated via upgrade.
      */
-    uint256 public immutable MIN_BOND_AMOUNT;
+    uint128 public immutable REQUIRED_BOND_AMOUNT;
```

Comment:
Why don't we use the unified type (`uint256`) for bond amount?
This is because the bond amount type of _increaseBond is uint256. In deposit(), msg.value is used as this bond amount. At this time, since msg.value is uint256, using uint256 for all bond amounts will prevent confusion.

Reply:
For https://fravoll.github.io/solidity-patterns/tight_variable_packing.html.
You can think of it as using uint256 as the unit for deposit and uint128 for bond.

Reply:
Got it thanks!